lib/ain-grpc/src/rpc/eth.rs

```diff
@@ -799,12 +802,12 @@ impl MetachainRPCServer for MetachainRPCModule {
                 if balance < value {
                     return Err(RPCError::InsufficientFunds.into());
                 }
-                available = balance.checked_sub(value).ok_or(RPCError::ValueOverflow)?;
+                available = balance.checked_sub(value).ok_or(RPCError::ValueUnderflow)?;
             }
 
             let allowance = available
                 .checked_div(fee_cap)
-                .ok_or(RPCError::ValueOverflow)?;
+                .ok_or(RPCError::DivideByZero)?;
```

Comment:
shouldn't validate the `fee_cap` is not zero before get divided for error `DividedByZero`??

Reply:
That is what `checked_div` does.

Reply:
hmm.. i mean that should have 2 validations:
i. fee_cap is not allowed to zero
ii. check_div for underoverflow checking

Reply:
Agree. And in relation to the other point, should be more generic. 

fee_cap check, ideal to have it checked, but probably not necessary if it isn't used directly later, and can just rely on checked_div